templates/desktop/index.html

```diff
@@ -28,121 +32,236 @@ <h1>Ubuntu for desktops</h1>
+    <div class="col">
+      <div class="row p-section--shallow">
+        <div class="col-3">
+          <h3 class="p-heading--2">Open source</h3>
+        </div>
+        <div class="col-6">
+          <p>Ubuntu has always been free to download, use and share.  We believe in the power of open source software; Ubuntu could not exist without its worldwide community of voluntary developers.</p>
+        </div>
+      </div>
+      <hr class="p-rule" />
+      <div class="row p-section--shallow">
+        <div class="col-3">
+          <h3 class="p-heading--2">Secure</h3>
+        </div>
+        <div class="col-6">
+          <p>With a built-in firewall and virus protection software, Ubuntu is one of the most secure operating systems around.  And the long-term support releases give you five years of security patches and updates.</p>
+        </div>
+      </div>
+      <hr class="p-rule" />
+      <div class="row p-section--shallow">
+        <div class="col-3">
+          <h3 class="p-heading--2">Accessible</h3>
+        </div>
+        <div class="col-6">
+          <p>Computing is for everyone regardless of nationality, gender or disability. Ubuntu is fully translated into over 50 languages and includes essential assistive technologies.</p>
+        </div>
+      </div>
     </div>
   </div>
 </section>
 
-<section class="p-strip is-bordered">
-  <div class="row u-equal-height">
-    <div class="col-6 u-hide--small u-hide--medium">
-      {{ image (
-        url="https://assets.ubuntu.com/v1/acdf946a-Screenshot+from+2022-04-18+13-05-17.png",
-        alt="",
-        width="2557",
-        height="1321",
-        hi_def=True,
-        loading="lazy"
-        ) | safe
-      }}
+<section class="p-section">
+  <div class="row--50-50 p-section--shallow">
+    <hr class="p-rule" />
+    <div class="col">
+      <h2>Visually stunning,<br /> wherever it's used</h2>
     </div>
-    <div class="col-6 u-vertically-center">
-      <div>
-        <h2>
-          Visually stunning, wherever it's used
-        </h2>
-        <p>
-          Ubuntu gets the most from your screen, with high definition, touchscreen support, fractional scaling and touchpad gestures. 22.04 refreshes its signature Yaru theme, with system-wide dark style preference support, accent colours and the largest selection of community wallpapers yet.
-        </p>
-      </div>
+    <div class="col">
+      <p>Ubuntu gets the most from your screen, with high definition, touchscreen support, fractional scaling and touchpad gestures. 22.04 refreshes its signature Yaru theme, with system-wide dark style preference support, accent colours and the largest selection of community wallpapers yet.</p>
     </div>
   </div>
+  <div class="u-fixed-width p-section--shallow">
+    {{ 
+      image (
+      url="https://assets.ubuntu.com/v1/acdf946a-Screenshot+from+2022-04-18+13-05-17.png",
+      alt="",
```

Comment:
I think it would be nice to have alt text on this image as well

Reply:
@akbarkz we don't typically add alt text to decorative images, from [w3](https://www.w3.org/WAI/tutorials/images/decorative/): "In these cases, a null (empty) alt text should be provided (alt="") so that they can be ignored by assistive technologies, such as screen readers. Text values for these types of images would add audible clutter to screen reader output or could distract users if the topic is different from that in adjacent text."